tools/utils/seed/template_locals.ts

```diff
@@ -1,21 +1,46 @@
+import * as util from 'gulp-util';
 import { argv } from 'yargs';
+import { join } from 'path';
+
 import * as CONFIG from '../../config';
 
+const getConfig = (path: string, env: string): any => {
+  const suffixPath = join(path, `${env}.config.js`);
+  const simplePath = join(path, `${env}.js`);
+  const config: any = [suffixPath, simplePath]
+    .reduce((config: any, path: string) => {
+      if (config) {
+        return config;
+      }
+      try {
+        config = require(path);
```

Comment:
@Minko:

I just checked it. I removed the `.js` suffix and ran `$ npm start` again, and it still errors out 🌵 

```bash
[15:33:49] Starting 'build.js.dev'...
[15:33:50] Cannot find /Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/env/base.config, neither /Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/env/base
[15:33:50] Cannot find /Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/env/dev.config, neither /Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/env/dev
[15:33:50] 'build.js.dev' errored after 815 ms
[15:33:50] Error: Invalid configuration name
    at Object.templateLocals (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/utils/seed/template_locals.ts:39:11)
    at module.exports (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/tasks/seed/build.js.dev.ts:61:28)
    at Gulp.<anonymous> (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/utils/seed/tasks_tools.ts:32:31)
    at module.exports (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/index.js:134:8)
    at runNextSet (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/run-sequence/index.js:86:16)
    at Gulp.onTaskEnd (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/run-sequence/index.js:75:5)
    at emitOne (events.js:101:20)
[15:33:50] 'serve.dev' errored after 1.49 s
[15:33:50] Error in plugin 'run-sequence(build.js.dev)'
Message:
    Invalid configuration name
Stack:
Error: Invalid configuration name
    at Object.templateLocals (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/utils/seed/template_locals.ts:39:11)
    at module.exports (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/tasks/seed/build.js.dev.ts:61:28)
    at Gulp.<anonymous> (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/utils/seed/tasks_tools.ts:32:31)
    at module.exports (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/index.js:134:8)
    at runNextSet (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/run-sequence/index.js:86:16)
    at Gulp.onTaskEnd (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/run-sequence/index.js:75:5)
    at emitOne (events.js:101:20)
[15:33:50] 'build.dev' errored after 1.49 s
[15:33:50] Error in plugin 'run-sequence(serve.dev)'
Message:
    Invalid configuration name
Stack:
Error: Invalid configuration name
    at Object.templateLocals (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/utils/seed/template_locals.ts:39:11)
    at module.exports (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/tasks/seed/build.js.dev.ts:61:28)
    at Gulp.<anonymous> (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/utils/seed/tasks_tools.ts:32:31)
    at module.exports (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/index.js:134:8)
    at runNextSet (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/run-sequence/index.js:86:16)
    at Gulp.onTaskEnd (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/run-sequence/index.js:75:5)
    at emitOne (events.js:101:20)
[15:33:50] 'build.dev' errored after 1.5 s
[15:33:50] Error in plugin 'run-sequence(build.js.dev)'
Message:
    Invalid configuration name
Stack:
Error: Invalid configuration name
    at Object.templateLocals (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/utils/seed/template_locals.ts:39:11)
    at module.exports (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/tasks/seed/build.js.dev.ts:61:28)
    at Gulp.<anonymous> (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/tools/utils/seed/tasks_tools.ts:32:31)
    at module.exports (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/orchestrator/index.js:134:8)
    at runNextSet (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/run-sequence/index.js:86:16)
    at Gulp.onTaskEnd (/Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/node_modules/run-sequence/index.js:75:5)
    at emitOne (events.js:101:20)

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/Cellar/node/6.3.1/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE
npm ERR! angular2-seed@0.0.0 start: `gulp serve.dev --color`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the angular2-seed@0.0.0 start script 'gulp serve.dev --color'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the angular2-seed package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp serve.dev --color
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs angular2-seed
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls angular2-seed
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/DonDope/Development/GitHub/TheDonDope/angular2-seed/npm-debug.log
```